R/begin.R

```diff
@@ -73,4 +75,12 @@ begin = function(
     easyr::runfolder( 'fun', verbose = verbose )
   }
 
-}
\ No newline at end of file
+  if( runpath != FALSE && dir.exists(runpath) ){
+    easyr::runfolder( runpath, verbose = verbose )
+  }
+  
+  if( runpath != FALSE && file_test('-f', runpath) ){
+    source(runpath, keep.source = FALSE)
+  }
```

Comment:
I think it would be cleaner to combine these 2 if statements. please use something like:
```
if(!is.null(runpath)){
  if(dir.exists(runpath)){
    runfolder(runpath, verbose = verbose)
  } else if(file.exists(runpath)){
    source(runpath, keep.source = FALSE)
  } else {
    stop(glue::glue('easyr begin error: runpath not found: [{runpath}].'))
  }
}
```